### PR TITLE
Fix error in touchmove event

### PR DESCRIPTION
**Pull request by maurispalletti**
Fix issue #128 .

e.cancelable validation added to e.preventDefault(), to avoid console error ```Ignored attempt to cancel a touchmove event with cancelable=false, for example because scrolling is in progress and cannot be interrupted.```.

### CODE REVIEW

File: src/__tests__/index.spec.js [hunk 1]
```diff
@@ -210,16 +211,17 @@ function setupGetMountedComponent(TYPE, mockListeners = mockListenersSetup) {
     it('calls preventDefault when onSwiping is present', () => {
       const onSwiping = jest.fn()
       const preventDefault = jest.fn()
+      const cancelable = jest.fn()
```
Comment (hartzis) [inline src/__tests__/index.spec.js]: 💡 mind updating this and the other test usage to `boolean`'s? `const cancelable = true`?

https://developer.mozilla.org/en-US/docs/Web/API/Event/cancelable

Comment (maurispalletti) [reply src/__tests__/index.spec.js]: Sure, my bad. I'll change it now and update the PR.

Comment (maurispalletti) [reply src/__tests__/index.spec.js]: PR updated!

**Comment by maurispalletti** [comment]
@hartzis, I haven't found any potential issues. It just helps to avoid that console error.
I think it this way: If the event is not cancellable, not checking cancellable prop will just throw an error because event.preventDefault() can't be executed. With this validation, event.preventDefault() won't be executed anyway, and without throwing an error.

**Comment by hartzis** [comment]
@maurispalletti I like your thinking! I'm also very comfortable with [browser support](https://developer.mozilla.org/en-US/docs/Web/API/Event/cancelable#Browser_compatibility).

Let's merge and publish this.

**Pull request merged**